diff --git a/servo/ports/geckolib/cbindgen.toml b/servo/ports/geckolib/cbindgen.toml
--- a/servo/ports/geckolib/cbindgen.toml
+++ b/servo/ports/geckolib/cbindgen.toml
@@ -1277,7 +1277,8 @@
   inline int8_t ShadowCascadeOrder() const {
     bool neg(*this & CASCADE_ORDER_SIGN);
     uint8_t abs = (*this & CASCADE_ORDER_BITS)._0 >> CASCADE_ORDER_SHIFT;
-    return neg ? -abs : abs;
+    return neg ? static_cast<int8_t>(-static_cast<int16_t>(abs))
+               : static_cast<int8_t>(abs);
   }
 """
 
