--- a/browser/app/profile/firefox.js
+++ b/browser/app/profile/firefox.js
@@ -1357,7 +1357,7 @@
 pref("browser.xul.error_pages.show_safe_browsing_details_on_load", false);
 
 // Enable captive portal detection.
-pref("network.captive-portal-service.enabled", true);
+pref("network.captive-portal-service.enabled", false);
 
 // If true, network link events will change the value of navigator.onLine
 pref("network.manage-offline-status", true);
@@ -2407,6 +2407,14 @@
 pref("browser.translations.quickAction.enabled", true);
 
 // Telemetry settings.
+pref("datareporting.healthreport.uploadEnabled", false);
+pref("datareporting.usage.uploadEnabled", false);
+pref("datareporting.policy.dataSubmissionEnabled", false);
+pref("datareporting.policy.dataSubmissionPolicyBypassNotification", true);
+pref("datareporting.policy.firstRunURL", "");
+pref("toolkit.telemetry.reportingpolicy.firstRun", false);
+pref("browser.region.network.url", "");
+pref("network.connectivity-service.enabled", false);
 // Determines if Telemetry pings can be archived locally.
 pref("toolkit.telemetry.archive.enabled", false);
 // Enables sending the shutdown ping when Firefox shuts down.
